docs/integrations.md

```diff
@@ -96,7 +69,46 @@ semgrep:
 
 </p>
 </details>
-</br>
+
+## Standalone providers
+
+Semgrep can also be manually integrated into CI pipelines using a fixed
+ruleset; however features of Semgrep Community like policies and notifications
+are unavailable.
+
+These can be configured for CircleCI or Buildkite as follows:
+
+<details><summary>Buildkite</summary>
```

Comment:
Can we note that deduplication will not work here? From that perspective a `Contact us` might be helpful.